### ISSUE TITLE
Test case fails

### ISSUE THREAD

**Issue by cymon** [Finn-Lab/SanntiS#1]
I note that this test case fails because line 66-67 in BGDetection.py is expecting 12 tab-sep fields whereas there are only 11 in the row returned by GeneID:

```
conda activate sanntis
sanntis files/BGC0001472.fna
...
$ tail -f err
Traceback (most recent call last):
  File "/share/apps/opt/miniconda3/envs/sanntis/bin/sanntis", line 10, in <module>
    sys.exit(main())
  File "/share/apps/opt/miniconda3/envs/sanntis/lib/python3.9/site-packages/sanntis/_cli.py", line 169, in main
    annotate.transformIPS(ips_file)
  File "/share/apps/opt/miniconda3/envs/sanntis/lib/python3.9/site-packages/sanntis/modules/BGCdetection.py", line 73, in transformIPS
    spl[11] if spl[11] != '-' else spl[4]
IndexError: list index out of range

```

**Comment by SantiagoSanchezF** [comment]
Hi @cymon,

The issue you're facing appears to be caused by a bug in InterProScan, which occasionally fails to return the expected number of fields in the TSV format. I managed to successfully execute the command using the most recent version of InterProScan (5.61-93.0). 

With the error handling in place, you should now be able to run the command without crashing, even if you are using an older version. However, I still recommend updating to InterProScan version 5.61-93.0 for optimal performance.

To resolve this issue, please update to the latest sanntis release (0.9.3.1, now available in bioconda).

Santiago.

**Comment by SantiagoSanchezF** [comment]
Sorry, this is fixed in 0.9.3.2
